modyn/tests/supervisor/internal/test_grpc_handler.py

```diff
@@ -98,8 +99,9 @@ def get_minimal_pipeline_config() -> dict:
 @patch.object(StorageStub, "__init__", noop_constructor_mock)
 @patch.object(EvaluatorStub, "__init__", noop_constructor_mock)
 @patch("modyn.supervisor.internal.grpc_handler.grpc_connection_established", return_value=True)
+@patch("modyn.common.grpc.grpc_helpers.grpc_connection_established", return_value=True)
 @patch.object(grpc, "insecure_channel", return_value=None)
-def get_non_connecting_handler(insecure_channel, init) -> GRPCHandler:
+def get_non_connecting_handler(*args) -> GRPCHandler:
```

Comment:
many Mocks but none of them are used in the function body. `*args` is to save some efforts.